### DISCUSSION TITLE
Migrate to Biome Formatter & Linter

### DISCUSSION THREAD

**Discussion by imsorx** [spacedriveapp/spacedrive#2721] [Feature Request]
Suggestion to use [Biome](https://biomejs.dev/). I've started using this for some projects at Work and works like a charm.

Pros
- Combines Formatter & Linter
- Compatible with Prettier & Eslint Configs
- Rust at core, hence FAST

**Comment by HeavenVolkoff** [comment]
Interesting project. Will take a look and try to use on some of our smaller repos, if it works well I will see if we can use it in the main spacedrive repo
